### PR TITLE
[Bug] Close the sectionPicker if the escape key is pressed

### PR DESCRIPTION
**Pull request by VishaalK**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ node_modules
 target/modules
 target/tests
 typings
+debug.log
```
Comment (leakymemory) [inline .gitignore:8]: What generates this?

File: src/scripts/UI/components/oneNotePickerComponent.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import {OneNotePickerPopupComponent} from "./oneNotePickerPopupComponent";
 import {SectionProps} from "./sectionComponent";
 import {ComponentBase} from "../componentBase";
 import {Status} from "../../status";
+import {Constants} from "../../constants";
```
Comment (leakymemory) [inline src/scripts/UI/components/oneNotePickerComponent.tsx]: Nit: We should order the imports.

File: target/oneNotePicker.js [hunk 3]
```diff
@@ -465,10 +466,34 @@ var OneNotePickerComponentClass = (function (_super) {
...
+            context.onunload = function () {
+                document.onkeydown = oldOnKeyDown_1 ? oldOnKeyDown_1.bind(document) : undefined;
+            };
+        }
+    };
     OneNotePickerComponentClass.prototype.render = function () {
         var status = this.getStatusEnumFromString(this.props.status);
         var textToDisplay = this.getTextToDisplayFromStatus(status);
-        return ({tag: "div", attrs: {}, children: [
+        // if (!OneNotePickerComponentClass.escapeListenerAttached) {
```
Comment (leakymemory) [inline target/oneNotePicker.js]: Is this code ready for review?

File: .gitignore (hunk 1 above)
Comment (VishaalK) [reply .gitignore:8]: node generates it. I think I ran some invalid command. These were the contents of mine. 
```
[0227/160250:ERROR:tcp_listen_socket.cc(76)] Could not bind socket to 127.0.0.1:6004
[0227/160250:ERROR:node_debugger.cc(86)] Cannot start debugger server
```

File: src/scripts/UI/components/oneNotePickerComponent.tsx (hunk 2 above)
Comment (VishaalK) [reply src/scripts/UI/components/oneNotePickerComponent.tsx]: Good catch!

Comment (VishaalK) [reply src/scripts/UI/components/oneNotePickerComponent.tsx]: Fixed

**Pull request merged**